### DISCUSSION TITLE
Distributing disks between nodes

### DISCUSSION THREAD

**Discussion by AlexZIX** [minio/minio#17814] [Q&A] [answered]
After few years of using minio I still can't understand how the disks in EC groups distributed between nodes. For ex. I have 6 nodes with 50 disks on each, totally 300 disks. The size of EC group will be 15. Is it possible that all disks of one EC group will be on the same node? Or it's guaranteed that each node will have 3 disks of each EC group?

**Comment by klauspost** [comment, marked as answer]
No. Sets are always distributed across servers first.

>  I have 6 nodes with 50 disks on each, totally 300 disks. The size of EC group will be 15. 

No. It will be 12, with 2 disks per set per server by default.

If you'd like a discussion beyond the [docs](https://min.io/docs/minio/linux/operations/concepts/erasure-coding.html) and [erasure calculator](https://min.io/product/erasure-code-calculator), feel free to reach out on `hello@min.io` and we can set up a meeting.
